### ISSUE TITLE
Class, interface or enum named `Symfony\Config\FrameworkConfig` does not exist

### ISSUE THREAD

**Issue by ThomasLandauer** [psalm/psalm-plugin-symfony#201]
With this being my `config/packages/mailer.php`:
```php
<?php declare(strict_types=1);

use Symfony\Config\FrameworkConfig;

return static function (FrameworkConfig $framework): void
{
    $framework->mailer()
        ->transport(...)
    ;
};
```
... psalm is reporting:
> ERROR: UndefinedClass - config/packages/mailer.php:5:25 - Class, interface or enum named Symfony\Config\FrameworkConfig does not exist (see https://psalm.dev/019)
return static function (FrameworkConfig $framework): void

This is essentially the same as https://github.com/symfony/symfony/issues/42342 for phptsan. The solution for phpstan was to add a hint to add `scanDirectories` to the config, see https://github.com/phpstan/phpstan-symfony/pull/190/files

Is something like that possible in psalm too? I tried this but it didn't work:
```xml
    <projectFiles>
        <directory name="var/cache/dev/Symfony/Config" />
    </projectFiles>
```

**Comment by seferov** [comment]
@ThomasLandauer thanks for the report.

Can you please try this?

```xml
<extraFiles>
    <directory name="var/cache/dev/Symfony/Config" />
</extraFiles>
```

**Comment by ThomasLandauer** [comment]
Thanks, works, see https://github.com/psalm/psalm-plugin-symfony/pull/203 :-)
